src/util/XMLUtils.h

```diff
@@ -21,7 +21,7 @@
  *
  */
 
-#include <platform/util/StdString.h>
+#include <p8-platform/util/StdString.h>
```

Comment:
In https://github.com/xbmc/kodi-platform/pull/13 , the change differs a bit:

``` diff
  *
   */
-#include <platform/util/StdString.h>
 +#include <util/StdString.h>
  #include "tinyxml.h"
```

Using `#include <util/StdString.h>` might be a cleaner solution, as long as it works.